.github/renovate.json5

```diff
@@ -12,16 +12,11 @@
   "separateMinorPatch": true,
   "packageRules": [
     {
-      "matchPackageNames": [
-        "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator",
-        "io.opentelemetry.proto:opentelemetry-proto",
-        "io.opentelemetry.semconv:opentelemetry-semconv"
-      ],
       // Renovate's default behavior is only to update from unstable -> unstable if it's for the
       // major.minor.patch, under the assumption that you would want to update to the stable version
       // of that release instead of the unstable version for a future release
-      // (TODO remove once the artifacts above release stable versions)
-      "ignoreUnstable": false
+      "matchCurrentVersion": "/^.*-alpha$/",
+      "allowedVersions": "/^.*-alpha$/"
```

Comment:
will this also auto-update from `-alpha` to a stable version?

Reply:
no, `allowedVersions` describes which versions are allowed as new versions, which is the regex `^.*-alpha$`